text_generation/causal_lm/cpp/README.md

```diff
@@ -112,6 +112,8 @@ convert_tokenizer .\TinyLlama-1.1B-Chat-v1.0\pytorch\dldt\FP16\ --output .\TinyL
 
 To enable Unicode characters for Windows cmd open `Region` settings from `Control panel`. `Administrative`->`Change system locale`->`Beta: Use Unicode UTF-8 for worldwide language support`->`OK`. Reboot.
 
+The suggestion is to run on dGPU (32GB+ RAM) models that work extremely slowly on CPU. For example, meta-llama/Llama-2-13b-chat-hf. Modify the source code to change the device for inference to GPU.
```

Comment:
looks strange to say in our samples that our devices are _extremely flow_
maybe we can reformulate it to denote that GPU provide better performance for 13B+ models?